packages/nimble-components/src/theme-provider/index.ts

```diff
@@ -33,19 +49,50 @@ export const theme = DesignToken.create<Theme>({
  * @internal
  */
 export class ThemeProvider extends FoundationElement {
-    @attr({
-        attribute: 'direction'
-    })
+    @attr()
+    public override lang = '';
```

Comment:
Had to add a TS suppression to do that. Leaving it uninitialized requires adding `undefined` to the type, but I can't do that because its type has to match the `lang` attribute that it's overriding. If I don't use the `override` keyword, that also produces an error.

Reply:
You could possibly use a non-null assertion

```ts
    @attr
    public override lang!;
```